server/views/partials/layout.njk

```diff
@@ -14,7 +14,5 @@
 {% endblock %}
 
 {% block bodyEnd %}
-  <script type="module" src="/assets/govukFrontendInit.js"></script>
```

Comment:
the design system docs [say that you "must" use `type="module"`](https://frontend.design-system.service.gov.uk/import-javascript/) though it's probably not _that_ hard a requirement… will our compiled app.js load ok as a module?